rotkehlchen/chain/bitcoin/__init__.py

```diff
@@ -13,65 +15,130 @@ def _have_bc1_accounts(accounts: List[BTCAddress]) -> bool:
     return any(account.lower()[0:3] == 'bc1' for account in accounts)
 
 
-def get_bitcoin_addresses_balances(accounts: List[BTCAddress]) -> Dict[BTCAddress, FVal]:
-    """Queries blockchain.info or blockstream for the balances of accounts
+def _query_blockstream_info(accounts: List[BTCAddress]) -> Dict[BTCAddress, FVal]:
+    """Queries balances from blockstream.info
+    May raise:
+    - RemoteError if got problems with querying the API
+    - KeyError if got unexpected json structure
+    - DeserializationError if got unexpected json values
+    """
+    balances = {}
+    for account in accounts:
+        url = f'https://blockstream.info/api/address/{account}'
+        response_data = request_get_dict(url=url, handle_429=True, backoff_in_seconds=4)
+        stats = response_data['chain_stats']
+        funded_txo_sum = deserialize_fval(value=stats['funded_txo_sum'], name='funded_txo_sum', location='blockstream api')  # noqa: E501
```

Comment:
Since you are going to change it already I advice you to change this. Use deserialize_int as I told you in the previous one. Since they are both ints and not fvals. And then when you get the funded_sum pass it to `satoshis_to_btc` since that accepts int and will return an `FVal`